Formula/pyinvoke.rb

```diff
@@ -0,0 +1,38 @@
+class Pyinvoke < Formula
+  include Language::Python::Virtualenv
+
+  desc "Pythonic task management & command execution"
+  homepage "http://pyinvoke.org/"
+  url "https://github.com/pyinvoke/invoke/archive/0.13.0.tar.gz"
+  sha256 "4942aa901618f99afa70271d7b15fd018da0f174178fbca4048354c785e7736a"
+
+  head "https://github.com/pyinvoke/invoke.git"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    (testpath/"tasks.py").write <<-EOS.undent
+      from invoke import run, task
+
+      @task
+      def clean(ctx, extra=''):
+          patterns = ['foo']
+          if extra:
+              patterns.append(extra)
+          for pattern in patterns:
+              run("rm -rf {}".format(pattern))
+    EOS
+    mkdir_p testpath/"foo"/"bar"
+    mkdir_p testpath/"baz"
+    system bin/"invoke", "clean"
+    assert !File.exist?(testpath/"foo"), "\"pyinvoke clean\" should have deleted \"foo\""
```

Comment:
Doesn't need to be decided right here, but now we're enforcing Ruby 2.0 we can use things like `assert_predicate` if we like, which produces nicer error messages by default without necessarily having to explicitly write out one.